PrizeDraw/PrizeDraw.csproj

```diff
@@ -8,11 +8,12 @@
     <OutputType>WinExe</OutputType>
     <RootNamespace>PrizeDraw</RootNamespace>
     <AssemblyName>PrizeDraw</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Isn't this downgrading, not upgrading?

Reply:
I've reapplied the upgrade and pushed again